src/strings.rs

```diff
@@ -424,3 +424,314 @@ fn message_contact_registered(sender: &MessageSender) -> String {
     let sender = message_sender(sender);
     gettext_f("{sender} joined Telegram", &[("sender", &sender)])
 }
+
+pub(crate) fn chat_action(action: &ChatAction) -> String {
```

Comment:
I want to port all of this away from the gettext macro before merging the PR. I'll comment here when I'll do it.

Reply:
Done https://github.com/melix99/telegrand/commit/23f77ced6b17c921166fa3a7675381f8b2ce1a47.

Reply:
I thought the process of moving away from gettext macro is already completed

Reply:
Looks like gettext! is still used in some places